tests/FunctionalTests/AbstractFunctionalTests.php

```diff
@@ -102,7 +111,7 @@ protected function assertStoredMessageIsInDB(Envelope $envelope): void
         /** @var MonitorIdStamp $monitorIdStamp */
         $monitorIdStamp = $envelope->last(MonitorIdStamp::class);
         $this->assertNotFalse(
-            $connection->executeQuery('SELECT id FROM messenger_monitor WHERE id = :id', ['id' => $monitorIdStamp->getId()])
+            $connection->executeQuery('SELECT id FROM messenger_monitor WHERE message_uid = :id', ['id' => $monitorIdStamp->getId()])
```

Comment:
pretty sure this was a bug in the tests that maria DB hid, but postgres exposed it. as  `$monitorIdStamp->getId()` is a uuid, which threw an error trying to coerse to an int